extension/test/unit/config/init/ensure-resources.spec.js

```diff
@@ -0,0 +1,64 @@
+const { expect } = require('chai')
+const sinon = require('sinon')
+const _ = require('lodash')
+const { ensureResources } = require('../../../../src/config/init/ensure-resources')
+
+const webComponentsPaymentType = require('../../../../resources/web-components-payment-type.json')
+const apiExtension = require('../../../../resources/api-extension.json')
+const interfaceInteractionType = require('../../../../resources/payment-interface-interaction-type.json')
+
+describe('Ensure resources', () => {
+  const mockClient = {
+    get builder () {
+      return {
+        extensions: {
+          where: () => {
+          }
+        },
+        types: {
+          where: () => {
+          }
+        }
+      }
+    },
+    fetch () {
+    },
+    create () {
+    }
+  }
+
+  afterEach(() => {
+    sinon.restore()
+  })
+
+  it('should ensure payment type, interface interaction and api extension are created', async () => {
```

Comment:
```suggestion
  it('should ensure payment and payment-interface-interaction types and api extension are created', async () => {
```

Reply:
`should ensure, **that**` I guess?

Reply:
You can have `that` but it is not compulsory :)  I guess we don't need it here if the meaning is clear: https://www.quickanddirtytips.com/education/grammar/when-to-delete-that

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/233/files#diff-f0a5acbf77903113e78b8ab9c34069e0R34